Brocfile.js

```diff
@@ -17,4 +18,13 @@ var app = new EmberApp();
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
-module.exports = app.toTree();
+// Foundation
+app.import('bower_components/foundation/js/foundation.min.js');
+
+var extraAssets = pickFiles('bower_components/fontawesome/fonts', {
```

Comment:
Also a name that explains what assets we have here would be helpful. Maybe something like `fontawesomeAssets`? Maybe we will have more of these collectors in the future and need to keep them apart.